### PR TITLE
SVCINT-1058 update slack result template visual

### PR DESCRIPTION
**Pull request by jbvouma**
[SVCINT-1058](https://coveord.atlassian.net/browse/SVCINT-1058)

Some updates in the Slack JSUI result template to match the changes asked in the ticket. 
Screen Shot 2021-08-18 at 3 58 20 PM

Final result
image

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

**Review by mikegron — CHANGES_REQUESTED** [review]

**Comment by mikegron** [comment]
Vraiment nice !!

File: src/ui/Templates/CoreHelpers.ts [hunk 1]
```diff
@@ -783,6 +783,8 @@ TemplateHelpers.registerTemplateHelper('isMobileDevice', () => {
   return DeviceUtils.isMobileDevice() ? DeviceUtils.getDeviceName() : null;
 });
 
+TemplateHelpers.registerTemplateHelper('pluralReplyHelper', (count: number) => (count == 1 ? 'reply' : 'replies'));
```
[the remark this answers is not in the document]
Comment (jbvouma) [reply src/ui/Templates/CoreHelpers.ts]: Done!

**Review by pathiery — APPROVED** [review]
Looks good ! Did you test to make sure the folding results ? Not sure what that comment is about ....

**Comment by jbvouma** [comment]
> Looks good ! Did you test to make sure the folding results ? Not sure what that comment is about ....

I just checked it, the [data-range](https://coveo.github.io/search-ui/classes/foldingforthread.html#options.range) set to 0 should do the trick.

**Review by mikegron — APPROVED** [review]

**Review by dguerinCoveo — APPROVED** [review]
Do you know why there was a different behaviour in dev and prod before you began?

**Comment by jbvouma** [comment]
> Do you know why there was a different behaviour in dev and prod before you began?

Not really but I was testing in the `coveo internal testing` org and let Denis know about it.

**Review by btaillon — APPROVED** [review]

**Pull request merged**